src/constants/categories.js

```diff
@@ -43,8 +43,8 @@ const categories:Array<Category> = [
         search: {
             q: "housing -(coordinating bodies) -(respite care) " +
                 "-(housing information) -hef",
-            service_type: "housing",
-            },
+                service_type: "housing",
```

Comment:
Does that mean the service types in everyday things & finding work are wrong?  I lined it up with them.